### DISCUSSION TITLE
icons are not displaying properly

### DISCUSSION THREAD

**Discussion by jjgitit** [NvChad/NvChad#2339] [Q&A] [answered]
I have downloaded nerd font, Jetbrain Mono NF, and set it as my iterm2 text style. However, when I open file explorer on NvChad, some icons are blank and those that appear seem to have weird size. Any help would be greatly appreciated as I'm trying to move from vscode to Nvim. 
![Screenshot 2023-08-25 at 11 52 31 PM](https://github.com/NvChad/NvChad/assets/132883866/22b0ac42-1d6b-411a-8075-7a64afef2255)

**Comment by siduck** [comment]
please set those in   your terminal or use a better terminal

**Comment by soomtong** [comment, marked as answer]
I've used iterm2 has a bug in the process of outputting icons. Instead, I recommend wezterm.

**Reply by jjgitit** [reply]
Thank you. I'll give it a try
